standalone/install.spec.ts

```diff
@@ -92,4 +92,20 @@ describe('Install', () => {
 			expect(config.doNotTrack).to.be.equal(doNotTrack);
 		});
 	});
+
+	describe('When skips install', () => {
+		let OLD_ENV = { ...process.env };
+
+		beforeEach(() => {
+			process.env.PACT_SKIP_BINARY_INSTALL = 'true';
+		});
+
+		afterEach(() => {
+			process.env = { ...OLD_ENV };
+		});
+
+		it.only('Should not download it', () => {
```

Comment:
I think this might be better as `it`, not `it.only`.

Reply:
My bad :face_palm:. Fixed 3efffc5c4bb08fe